grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -361,26 +360,18 @@ class StockMovementApiController {
         render([data: pendingRequisitionDetails] as JSON)
     }
 
-    private Date parseDateRequested(String date) {
-        return DateUtil.asDate(date, Constants.EXPIRATION_DATE_FORMATTER)
-    }
-
-    private Date parseDateShipped(String date) {
-        return DateUtil.asDate(date, Constants.DELIVERY_DATE_FORMATTER)
```

Comment:
this is the real change. Notice that before we were providing the specific formatter to use but now we don't. Now this will use the flexible date formatter which allows for a much wider range of possible inputs